share/spice/dogo_news/dogo_news.js

```diff
@@ -0,0 +1,50 @@
+(function(env) {
+    "use strict";
+
+    env.ddg_spice_dogo_news = function(api_result) {
+        if (!api_result || !api_result.results || !api_result.results.length) {
+            return Spice.failed('dogo_news');
+        }
+
+        // Get original query.
+        var script = $('[src*="/js/spice/dogo_news/"]')[0],
+            source = $(script).attr("src"),
+            query = decodeURIComponent(source.match(/dogo_news\/([^\/]+)/)[1]);
+    
+        Spice.add({
```

Comment:
With your code above, I'm getting an error with ```moment```:

```Uncaught ReferenceError: moment is not defined```

What do I need to do to get access to the moment helper?